deployments/EKS/kubearmor.yaml

```diff
@@ -23,312 +25,322 @@ metadata:
   name: kubearmor
   namespace: kube-system
 spec:
-  selector:
-    kubearmor-app: kubearmor-relay
   ports:
   - port: 32767
     protocol: TCP
+    targetPort: 32767
+  selector:
+    kubearmor-app: kubearmor-relay
+status:
+  loadBalancer: {}
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: kubearmor-relay
-  namespace: kube-system
   labels:
     kubearmor-app: kubearmor-relay
+  name: kubearmor-relay
+  namespace: kube-system
 spec:
   replicas: 1
   selector:
     matchLabels:
       kubearmor-app: kubearmor-relay
+  strategy: {}
   template:
     metadata:
-      labels:
-        kubearmor-app: kubearmor-relay
       annotations:
         kubearmor-policy: audited
+      labels:
+        kubearmor-app: kubearmor-relay
     spec:
-      serviceAccountName: kubearmor
-      nodeSelector:
-        kubernetes.io/os: linux
       containers:
-      - name: kubearmor-relay-server
-        image: kubearmor/kubearmor-relay-server:latest
-        imagePullPolicy: Always
+      - image: kubearmor/kubearmor-relay-server:latest
+        name: kubearmor-relay-server
         ports:
         - containerPort: 32767
         resources:
-          requests:
-            cpu: 250m
-            memory: 64Mi
           limits:
             cpu: 500m
             memory: 128Mi
+          requests:
```

Comment:
We decided to remove resource limits, didn't we?

Reply:
I only removed limits for the Daemonset since only that caused the crashes!
Do we remove limits from other deployments as well?

Reply:
I think it would be better to remove the resource limits for the relay server too
since I feel it could also require the resources more than we expected in the case that we have a large number of nodes.